monero-outreach-docs/translations/es/archive/rpc_pay_es.md

```diff
@@ -1,113 +1,109 @@
 
-RPC-Pay is private. No large credit balances are maintained that require sharing information for backup or security. The mining hashes used for payment are anonymous without a way to trace ownership on the blockchain. A payment ID can be applied to track and maintain balances, but with an operating procedure based on small balances, this ID can be changed regularly or not used at all.
+Es importante también considerar las desventajas en la implementación del RPC-Pay en el actual _monerod_ y el monedero. Puede ser una carga a clientes con baja potencia, tal como los dispositivos móviles quienes deben apoyarse de la ayuda de otros dispositivos para sus hashes. Además, las ganancias al servidor son aleatorias. La ganancia solo viene cuando el cliente suministra un hash que logra minar un bloque, mientras que la salida del servidor debe ser continuo, igualando la provisión regular de los hashes. Finalmente, la cantidad de ganancias que un servidor puede hacer está limitado por el cronograma de la minería de Monero y tenderá a ser pequeña.
 
-It is important to also consider the disadvantages of the current _monerod_/wallet implementation of RPC-Pay. It can burden low-power clients, such as mobile devices, which must rely on other helper computers for their hashes. Also, the income to the server is random. It comes only when a client provides a hash that can mine a block, while the server’s output must be continuous, matching the regular provision of hashes. Finally, the amount of income a server can make is limited by the Monero mining schedule and will tend to be small.
-
-There is more to RPC-Pay that just Monero programs paying other Monero programs, as any server can use RPC-Pay to earn income. The software in the official Monero node and wallet is open source and can be reused for innovative applications, or other applications can communicate with monerod instances. The core concept is simply that the server gives its Monero address to its clients and the clients use this to create Monero mining hashes that it sends to the server as payment. [Example #3](https://www.monerooutreach.org/stories/RPC-Pay.html#box3) gives an example of RPC-Pay software source code.
+Hay más de RPC-Pay que solo programas de Monero pagando a otros programas del mismo Monero, ya que cualquier servidor puede usar RPC-Pay para tener ganancia. El software oficial de Monero en el nodo y el monedero es de código abierto y puede ser reutilizado para servicios innovadores, o para que otras aplicaciones puedan comunicarse con las instancias del monerod. El concepto central simplemente es que el servidor le da su propia dirección de Monero a sus clientes y estos la usan para crear hashes enviandolos al servidor como pago en forma de minería. El [Ejemplo #3](https://www.monerooutreach.org/stories/RPC-Pay.html#box3) da un ejemplo del código del software de RPC-Pay.
```

Comment:
A little re-arrangement:

_El software **en el nodo de Monero oficial** y el monedero..._

And a correction:

_...y estos la usan para crear hashes **enviándolos* al servidor como pago en forma de minería._

Reply:
Re-arrangement and correction added.